tools/generate_sysroot.py

```diff
@@ -113,6 +113,14 @@
   'wayland-devel',
 ]
 
+# Only available for Tizen 5.5 and above.
+autofill_packages = [
```

Comment:
The embedder is already built for Tizen 5.5+. No need to separate these packages. The only exceptions are Dali2 packages which are only available for Tizen 6.5+.

Reply:
Thank you. Removed unnecessary package separation.